src/ApiPort.VisualStudio/ApiPortVSPackage.cs

```diff
@@ -100,8 +100,12 @@ protected override void Initialize()
             }
         }
 
-        public void ShowToolbar()
+        public async System.Threading.Tasks.Task ShowToolbarAsync()
         {
+            // Calls to UI elements should use the main task thread.
+            // https://blogs.msdn.microsoft.com/andrewarnottms/2014/05/07/asynchronous-and-multithreaded-programming-within-vs-using-the-joinabletaskfactory/
+            await ThreadHelper.JoinableTaskFactory.SwitchToMainThreadAsync();
```

Comment:
This method is called from the ProjectAnalyzer after analysis, which may not be on the main thread... I wanted to ensure we were always on it.